doc/SYSTEM_DEPENDENCIES

```diff
@@ -5,7 +5,7 @@ postgresql
 libpq-dev
 build-essential
 libffi-dev
-python3-dev
+python3.5-dev
```

Comment:
What's this change for? (The last version of Ubuntu to include Python 3.5 was 16.04...)